Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,19 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  // eslint-disable-next-line prettier/prettier
+  return tasks
+    .map(task => task.duration / 60.0)
+    .filter(durationInHour => durationInHour >= 2.0)
+    .reduce((totalEarning, hourlyEarning) => totalEarning + hourlyEarning * hourlyRate, 0);
```

Comment:
`hourlyEarning` is actually still a duration. So I would keep the name `durationInHour`.

Reply:
Yes, you are absolutely right! Thank you, I fixed the naming.